cognite/client/_api/datapoint_tasks.py

```diff
@@ -255,8 +261,6 @@ def _verify_options_and_categorize_query(query: DatapointsQuery) -> bool:
             raise ValueError("When passing `aggregates`, argument `granularity` is also required.")
         elif query.include_outside_points is True:
             raise ValueError("'Include outside points' is not supported for aggregates.")
-        elif query.include_status is True:
-            raise ValueError("'Include status' is not supported for aggregates.")
```

Comment:
just for sanity check, there was no negative test for this case before?

Reply:
Seems so